src/localization/en.js

```diff
@@ -1,13 +1,13 @@
 export default {
 
-  langName: 'English',
+  langname: 'English',
```

Comment:
I feel like I must have asked this before, but why was this changed away from camel case?

Reply:
Oops, that was definitely a mistake. It got caught in my find and replace for `name: "xxx"` fields. Will fix now.